Sources/NIOWebSocket/WebSocketFrame.swift

```diff
@@ -71,6 +72,27 @@ extension WebSocketMaskingKey: ExpressibleByArrayLiteral {
     }
 }
 
+extension WebSocketMaskingKey {
+    /// Returns a random masking key, using the given generator as a source for randomness.
+    /// - Parameter generator: The random number generator to use when creating the
+    ///     new random masking key.
+    /// - Returns: A random masking key
+    @inlinable
+    public static func random<T>(
+        using generator: inout T
+    ) -> Self where T: RandomNumberGenerator {
```

Comment:
```suggestion
    public static func random<Generator: RandomNumberGenerator>(
        using generator: inout Generator
    ) -> WebSocketFrame {
```

- `Generator` is a better name than `T`
- `Self` won't work here for Swift 5.0 (IIRC)

Reply:
I fully agree that Generator is a better name. However, the standard library uses `T` consistently in its API for example in [FixedWidthInteger](https://github.com/apple/swift/blob/a5440a8abd89336ad1650a58266222d30f86d7c5/stdlib/public/core/Integers.swift#L2661-L2664).
I chose the name `T` to be more in line with the Standard library but I can still change it if you want :)

Reply:
In most other places (eg. Optional/Array/...) the stdlib actually uses nicer names so I don't think we should make our API worse because (I'm guessing?) the stdlib team missed the `T` here? @Lukasa wdyt? If we prefer shorter names for type parameters we could do an `RNG: RandomNumberGenerator` maybe?

This isn't blocking the merge ofc

Reply:
Agreed, I think using longer generic type names is generally better.